src/core/Akka.FSharp/FsApi.fs

```diff
@@ -481,6 +480,74 @@ let asyncReceive (i : Inbox) : Async<'Message option> =
     }
 
 /// <summary>
-/// Orders inbox to watch an actor targeted by provided <paramref name="actorRef"/>.
+/// Orders a <paramref name="watcher"/> to monitor an actor targeted by provided <paramref name="subject"/>.
+/// When an actor refered by subject dies, a watcher should receive a <see cref="Terminated"/> message.
+/// </summary>
+let monitor (subject: ActorRef) (watcher: ICanWatch) : ActorRef = watcher.Watch subject
+
+/// <summary>
+/// Orders a <paramref name="watcher"/> to stop monitoring an actor refered by provided <paramref name="subject"/>.
+/// </summary>
+let demonitor (subject: ActorRef) (watcher: ICanWatch) : ActorRef = watcher.Unwatch subject
+
+/// <summary>
+/// Subscribes an actor reference to target channel of the provided event stream.
+/// </summary>
+let sub (channel: System.Type) (ref: ActorRef) (eventStream: Akka.Event.EventStream) : bool = eventStream.Subscribe(ref, channel)
+
+/// <summary>
+/// Unubscribes an actor reference from target channel of the provided event stream.
+/// </summary>
+let unsub (channel: System.Type) (ref: ActorRef) (eventStream: Akka.Event.EventStream) : bool = eventStream.Unsubscribe(ref, channel)
+
+/// <summary>
+/// Publishes an event on the provided event stream. Event channel is resolved from event's type.
+/// </summary>
+let pub (event: 'Event) (eventStream: Akka.Event.EventStream) : unit = eventStream.Publish event
```

Comment:
This kind of abbreviation is not normal in F# coding. There are some exceptions - e.g. the F# library function "List.iter", or "obj", but these are rare and in an upstack framework we wouldn't expect to see abbreviations like this. Just use a full name unless there's a really, really good reason not to.